museum/sampleRecords/itemData.destination.json

```diff
@@ -0,0 +1,362 @@
+      "ca_objects.MakersMarks":"Harper, Mary Alison",
+      "ca_objects.ItemDates":"c1974",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Oil on Masonite",
+      "ca_objects.Dimensions.DimensionsValue":"H: 45.4cms W: 55.5cms.  Unframed.",
+      "classificationHierarchy":"ILLUSTRATIONS;Oil;building"
+    },
+    {
+      "object_id":"3",
+      "id":"3",
+      "idno":"A1964.26a",
+      "display_label":"TOWN HALL PERTH, ETCHING",
+      "ca_objects.Description":"Brown wood with gilt line frame, glazed, engraving\/ etching of Perth Town Hall clock tower &amp; people on Hay Street corner, painted c1916. Signed LL.",
+      "ca_objects.MakersMarks":"Van Raalte, Henri Benedictus (HenrI)",
+      "ca_objects.ItemDates":"1916",
+      "ca_objects.EarliestYear":"1916",
+      "ca_objects.LatestYear":"1916",
+      "ca_objects.HistoricalDetails":"Henri Van Raalte 1881-1929.   Refer notes D. Erickson's Audit 2014",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Etching on paper",
+      "ca_objects.Dimensions.DimensionsValue":"H: 39.3cms W: 21cms. Frame:",
+      "classificationHierarchy":"Root node for Classification;ILLUSTRATIONS;Etching;building"
+    },
+    {
+      "object_id":"4",
+      "id":"4",
+      "idno":"A1964.26b",
+      "display_label":"BROKEN WALLS, ETCHING",
+      "ca_objects.Description":"Brown wood with gilt line frame; glazed, engraving\/etching of old wooden buildings with Perth Town Hall clock tower in background; seen though tall brick doorway. Signed LR.",
+      "ca_objects.MakersMarks":"van Raalte, Henri Benedictus (HenrI)",
+      "ca_objects.ItemDates":"1917-1919",
+      "ca_objects.EarliestYear":"1917",
+      "ca_objects.LatestYear":"1919",
+      "ca_objects.HistoricalDetails":"Henri Van Raalte b.1881-1929",
+      "ca_objects.PhysicalCondition":"2",
+      "ca_objects.Materials":"Etching on paper",
+      "ca_objects.Dimensions.DimensionsValue":"H: 35.5cm W: 13.7cm. Frame:",
+      "classificationHierarchy":"Root node for Classification;ILLUSTRATIONS;Etching;building"
+    },
+    {
+      "object_id":"5",
+      "id":"5",
+      "idno":"P1999\/1006",
+      "display_label":"COLONIAL HOSPITAL, ST MARY'S CATHEDRAL &amp; R.C. BISHOPS HOUSE",
+      "ca_objects.Description":"Colonial Hospital, St Mary's Cathedral &amp; Roman Catholic Bishops House. Victoria Square c1860s\n\nSite of Royal Perth Hospital",
+      "ca_objects.MakersMarks":"Unknown",
+      "ca_objects.ItemDates":"c1863",
+      "ca_objects.EarliestYear":"1863",
+      "ca_objects.LatestYear":"1863",
+      "ca_objects.SignificanceCriteria":"Historic;Social or Spiritual",
+      "ca_objects.StatementOfSignificance":"Historic: \nSocial:",
+      "ca_objects.Dimensions.DimensionsValue":"Large size 28x22 cm",
+      "classificationHierarchy":"Building;hospitals"
```

Comment:
This appears to be a cacheing issue in the browse web service. When querying for this single record, the `classificationHierarchy` comes out correct.
``` json
{
  "ok":true,
  "results":[
    {
      "object_id":"393",
      "id":"393",
      "idno":"P1999\/1006",
      "display_label":"COLONIAL HOSPITAL, ST MARY'S CATHEDRAL &amp; R.C. BISHOPS HOUSE",
      "ca_objects.Description":"Colonial Hospital, St Mary's Cathedral &amp; Roman Catholic Bishops House. Victoria Square c1860s\n\nSite of Royal Perth Hospital",
      "ca_objects.MakersMarks":"Unknown",
      "ca_objects.ItemDates":"c1863",
      "ca_objects.EarliestYear":"1863",
      "ca_objects.LatestYear":"1863",
      "ca_objects.SignificanceCriteria":"Historic;Social or Spiritual",
      "ca_objects.StatementOfSignificance":"Historic: \nSocial:",
      "ca_objects.Dimensions.DimensionsValue":"Large size 28x22 cm",
      "classificationHierarchy":"Root node for Classification;PLACE;Building;hospitals"
    }
  ]
}
```